<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartCalc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="result_line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border-bottom: 1px solid gray;
}

background-color : white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="buddy">
     <cstring>result_line</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_unar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 151, 57);
	color: white;
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #FF7832, stop: 1 #FF9739)
}
</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}

</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}

</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mul">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 151, 57);
	color: white;
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #FF7832, stop: 1 #FF9739)
}
</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}

</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}

</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}

</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sub">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 151, 57);
	color: white;
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #FF7832, stop: 1 #FF9739)
}
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}

</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}

</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}

</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sum">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 151, 57);
	color: white;
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #FF7832, stop: 1 #FF9739)
}
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}

</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #dadbde, stop: 1 #f6f7fa)
}

</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 151, 57);
	color: white;
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #FF7832, stop: 1 #FF9739)
}
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_point">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>400</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open_br">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close_br">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_extent">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PIC">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>Pic</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_X">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_X">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>73</y>
      <width>175</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_C">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_FIN">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>FIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	border: 1px solid gray
}
QPushButton:pressed {
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
								stop: 0 #BEBEBE, stop: 1 #D7D7D7)
}
</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
